<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drill_editchart_options_basedialog</class>
 <widget class="QDialog" name="drill_editchart_options_basedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>229</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit Chart Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="pushButtonOk">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>9</y>
     <width>42</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidgetOptions">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>9</y>
     <width>336</width>
     <height>207</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabViewport">
    <attribute name="title">
     <string>Viewport</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>9</y>
       <width>310</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>X axis</string>
     </property>
     <widget class="QComboBox" name="comboBoxScaleX">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>156</width>
        <height>19</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Scale type: Linear or Logarithmic</string>
      </property>
      <item>
       <property name="text">
        <string>Linear</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Logarithmic</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="textLabelLowX_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>53</y>
        <width>45</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Scale:</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLowX">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>21</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Enter lowest value to display on the X axis</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditHighX">
      <property name="geometry">
       <rect>
        <x>218</x>
        <y>21</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Enter highest value to display on the X axis</string>
      </property>
     </widget>
     <widget class="QLabel" name="textLabelHighX">
      <property name="geometry">
       <rect>
        <x>167</x>
        <y>22</y>
        <width>45</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>High X:</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabelLowX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>22</y>
        <width>45</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Low X:</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>93</y>
       <width>310</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Y axis</string>
     </property>
     <widget class="QComboBox" name="comboBoxScaleY">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>156</width>
        <height>19</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Scale type: Linear or Logarithmic</string>
      </property>
      <item>
       <property name="text">
        <string>Linear</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Logarithmic</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="textLabelHighY">
      <property name="geometry">
       <rect>
        <x>167</x>
        <y>22</y>
        <width>45</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>High Y:</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabelLowY">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>22</y>
        <width>45</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Low Y:</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="textLabelLowX_2_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>53</y>
        <width>45</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Scale:</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditLowY">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>21</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Enter lowest value to display on the Y axis</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditHighY">
      <property name="geometry">
       <rect>
        <x>218</x>
        <y>21</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Enter highest value to display on the Y axis</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabTitles">
    <attribute name="title">
     <string>Titles</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxChartTitle">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>9</y>
       <width>318</width>
       <height>50</height>
      </rect>
     </property>
     <property name="title">
      <string>Chart title</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <widget class="QLineEdit" name="lineEditChartTitle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>295</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Enter Chart's title</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxChartLegendX">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>63</y>
       <width>318</width>
       <height>50</height>
      </rect>
     </property>
     <property name="title">
      <string>X-Axis legend</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <widget class="QLineEdit" name="lineEditAxisLegendX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>295</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Text legend for X axis</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxChartLegendY">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>119</y>
       <width>318</width>
       <height>50</height>
      </rect>
     </property>
     <property name="title">
      <string>Y-Axis legend</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <widget class="QLineEdit" name="lineEditAxisLegendY">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>295</width>
        <height>20</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Text legend for Y axis</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>lineEditChartTitle</tabstop>
  <tabstop>lineEditAxisLegendX</tabstop>
  <tabstop>lineEditAxisLegendY</tabstop>
  <tabstop>lineEditLowX</tabstop>
  <tabstop>lineEditHighX</tabstop>
  <tabstop>lineEditLowY</tabstop>
  <tabstop>lineEditHighY</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>tabWidgetOptions</tabstop>
 </tabstops>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
